test/ussd_popi_faq.test.js

```diff
@@ -0,0 +1,597 @@
+var vumigo = require("vumigo_v02");
+var AppTester = vumigo.AppTester;
+var assert = require('assert');
+var fixtures_IdentityStore = require('./fixtures_identity_store');
+var fixtures_StageBasedMessaging = require('./fixtures_stage_based_messaging');
+var fixtures_MessageSender = require('./fixtures_message_sender');
+var fixtures_Hub = require('./fixtures_hub');
+var fixtures_Jembi = require('./fixtures_jembi');
+var fixtures_ServiceRating = require('./fixtures_service_rating');
```

Comment:
Take a look at these fixtures and make sure you need all of them. I suspect there are a few that can be removed